src/antsibull_changelog/yaml.py

```diff
@@ -15,16 +15,21 @@
 import yaml
 
 _SafeLoader: Any
-_SafeDumper: Any
 try:
     # use C version if possible for speedup
-    from yaml import CSafeDumper as _SafeDumper
     from yaml import CSafeLoader as _SafeLoader
 except ImportError:
-    from yaml import SafeDumper as _SafeDumper
     from yaml import SafeLoader as _SafeLoader
 
 
+class _SafeDumper(yaml.SafeDumper):
```

Comment:
Is CSafeDumper not compatible with the `increase_indent` override?

Reply:
I actually tried that but it doesn't seem to support it, presumably because it's implemented in C it doesn't allow Python-style method overrides and overriding methods would require the C implementation to explicitly provide mechanisms for it.

Reply:
But I'm all ears if you figure out how to do it with the CSafeDumper :)